### PR TITLE
Xsup 26505/cisco sma fix filter message search command

### PR DESCRIPTION
**Pull request by MosheEichler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-26505

## Description
Added the `quote_params_via` parameter in the `_http_request` function in `CommionServerPython` in order to allow specific decoding for query params. by default spaces replaced with `+` using  the new `quote_params_via` parameters we can choose to replace with `%20`.
For more info see: https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlencode.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CiscoSMA/ReleaseNotes/1_1_16.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Cisco Security Management Appliance
+Fixed an issue where the ***cisco-sma-message-search*** command with **subject_filter_operatorwould** argument did not filter the message correctly.
```
Comment (ShirleyDenkberg) [inline Packs/CiscoSMA/ReleaseNotes/1_1_16.md:4]:
```suggestion
Fixed an issue where the ***cisco-sma-message-search*** command with the **subject_filter_operatorwould** argument did not filter the message correctly.
```

**Comment by ShirleyDenkberg** [comment]
@MosheEichler Doc review completed.

**Review by bziser — DISMISSED** [review]

**Comment by tkatzir** [comment]
Please don't merge until we verify this really fixes the issue.
FYI @DeanArbel, @bziser

**Review by bziser — APPROVED** [review]
great job!

**Comment by bziser** [comment]
Please update the description of the PR

**Pull request closed without merging**